android/build.gradle

```diff
@@ -36,10 +47,18 @@ dependencies {
   implementation 'com.android.support:appcompat-v7:28.0.0'
   implementation "com.google.android.gms:play-services-wallet:$googlePlayServicesVersion"
   implementation "com.google.firebase:firebase-core:$firebaseVersion"
-  implementation 'com.stripe:stripe-android:9.3.5'
+  implementation 'com.stripe:stripe-android:10.1.1'
   implementation 'com.github.tipsi:CreditCardEntry:1.5.1'
 }
-
 repositories {
   google()
+  mavenCentral()
+  maven {
+    url "https://jitpack.io"
+  }
+  jcenter()
+  maven {
+    // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
+    url "$rootDir/../example/node_modules/react-native/android"
```

Comment:
@mindlapse -- this is in the root android/build.gradle, but appears to reference the example's node_modules -- I see a problem here!

Reply:
These are some Quality of Life changes to make it easy to set up the project locally.  I am going to look closer at this one, and may remove the changes if they interfere

Reply:
I think this one in particular is a problem, because it will affect downstream projects when they pull this module into their projects. The problem could go away if you rewrite this line when publishing to NPM however. (Long term that sounds useful -- if we're willing to do that, we might be able to migrate the JS to TypeScript but publish JS to npm), but today this seems premature.

Reply:
(Unless I'm wrong, of course!)